src/event-reduce.ts

```diff
@@ -112,6 +125,48 @@ export function getQueryParams<RxDocType>(
     );
 }
 
+// This catches a specific case where we have a limit query (of say LIMIT items), and then
```

Comment:
ooc have you tested to confirm the limit buffer stuff works? i guess we have tests for it

Reply:
I did not test anything tbh. I just rebased everything we h ad into a commit and resolved the merge conflicts. I think this comment is from last year :-).

Reply:
ya it is. just seeing the limit buffer code reminded me that we need to make sure that still works. it's all my custom code to make rxdb faster

Reply:
Yeah I can test more, there was nothing obvious that broke so far though and the code is/was pretty similar still in rx-query (and therefore I somewhat assume event-reduce to behave similar).